src/eye-dome.wgsl

```diff
@@ -31,6 +33,6 @@ fn fragment(
     response += max(0.0, log_depth - log2(textureLoad(input_texture, vec2<i32>(ilocation.x, ilocation.y - 1), 0)));
     response /= 4.0;
 
-    var shade = exp(-response * 300.0 * 1.0);
+    var shade = exp(-response * 300.0 * edl_strength);
```

Comment:
What's the reason for `300.0` magic number? Can we bake that into `edl_strength` now?

Reply:
This came from the original potree paper. They said "The factor 300 is an empirically
determined value for a suitable base shading strength."